rascal2/ui/presenter.py

```diff
@@ -23,8 +31,61 @@ def createProject(self, name: str, save_path: str):
             The name of the project.
         save_path : str
             The save path of the project.
+
         """
 
         self.model.createProject(name, save_path)
         # TODO if the view's central widget is the startup one then setup MDI else reset the widgets.
         self.view.setupMDI()
+
+    def getControlsAttribute(self, setting) -> Any:
```

Comment:
if it's here then the FitSettingsWidget (which is a view) does not need to directly talk to the model, which is what we want from MVP surely? in any case i think it may be useful for the plotting widget to know whether the thing being run is Bayesian or not, which could easily be grabbed from the `procedure`

Reply:
This is okay along as you are just getting state what should never happen is the view manipulating (changing the state of) the model. If we had to make a getter in the presenter for everything we need from the model then the presenter might consist largely of getters which might be fine for C++ but not great for Python.